src/Connector.sol

```diff
@@ -139,15 +150,33 @@ contract CentrifugeConnector is Test {
         memberlist.updateMember(user, validUntil);
     }
 
-    function transferTo(
+    function handleTransfer(
         uint64 poolId,
         bytes16 trancheId,
         address user,
         uint256 amount
     ) public onlyRouter {
         RestrictedTokenLike token = RestrictedTokenLike(tranches[poolId][trancheId].token);
+        require(address(token) != address(0), "CentrifugeConnector/unknown-token");
         require(token.hasMember(user), "CentrifugeConnector/not-a-member");
         token.mint(user, amount);
     }
-    
+
+    function transfer(
+        uint64 poolId,
+        bytes16 trancheId,
+        address user,
+        uint256 amount,
+        string memory domainName
+    ) public {
+        uint32 domainId = domainLookup[keccak256(bytes(domainName))];
```

Comment:
Is it an idea that we create a function to lookup a domain given a domain name? To avoid duplicating this lookup impl logic every time we look it up